rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,138 @@
+### DCP PR:
+***Leave this blank until the RFC is approved then the Author(s) must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc\#\](https://github.com/HumanCellAtlas/dcp-community/pull/\<PR\#\>)`
+
+# User Support SOP
+
+## Summary
+
+This process RFC describes how the DCP handles user support requests through the use of the Zendesk application and other processes.
+
+## Author(s)
```

Comment:
The ` needs to be removed from the addresses below.

Reply:
done